.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,25 @@
+# code created with help of the github documentation : https://docs.github.com/en/actions/guides/publishing-docker-images
+
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+      
+  build-and-push:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v2
+        with:
```

Comment:
You are using the action from: https://github.com/marketplace/actions/build-and-push-docker-images yet you are using the arguments from a different action: https://github.com/marketplace/actions/docker-build-push-action (an unofficial one). This results in the following error in the workflow:
![Screenshot from 2021-04-21 10-41-35](https://user-images.githubusercontent.com/1296726/115524443-6e07e900-a28e-11eb-88f0-e89baed617ae.png)

Reply:
Thanks for the review! I had to mix the two solutions before because the secrets had not been set in the repository, I fell back to the official action now.